quicfire_tools/inputs.py

```diff
@@ -621,28 +636,184 @@ def set_output_files(
         self.quic_fire.surf_eng_out = int(surf_eng)
         self.quic_fire.emissions_out = 2 if emissions else 0
 
+    def new_wind_sensor(
+        self,
+        update: Optional[str] = None,
+        wind_speeds: Optional[Union[PositiveFloat, list(PositiveFloat)]] = None,
+        wind_directions: Optional[Union[NonNegativeInt, list(NonNegativeInt)]] = None,
+        wind_times: Optional[Union[NonNegativeInt, list(NonNegativeInt)]] = None,
+        sensor_height: Optional[PositiveFloat] = None,
+        x_location: Optional[PositiveInt] = None,
+        y_location: Optional[PositiveInt] = None,
```

Comment:
SimulationInputs isn't a Pydantic BaseModel, and we're in Python >= 3.9 land so we can do away with the Optional and Union type annotations.